garrysmod/lua/menu/errors.lua

```diff
@@ -55,9 +55,12 @@ end )
 
 local matAlert = Material( "icon16/error.png" )
 
+local cl_drawhud = GetConVar("cl_drawhud")
+
 hook.Add( "DrawOverlay", "MenuDrawLuaErrors", function()
 
 	if ( table.IsEmpty( Errors ) ) then return end
+	if (not cl_drawhud:GetBool() ) then return end

```

Comment:
if ( !cl_drawhud:GetBool() ) then

Needs the space and use C ops.

Reply:
Done